Sources/YBottomSheet/BottomSheetController.swift

```diff
@@ -392,6 +384,6 @@ internal extension BottomSheetController {
 
     @objc
     func simulateDismiss() {
```

Comment:
```suggestion
    func simulateTapCloseButton() {
```
Let's rename this to be more specific now that it has more specific behavior.